packages/typespec-vscode/src/typespec-utils.ts

```diff
@@ -53,3 +53,13 @@ export async function TraverseMainTspFileInWorkspace() {
         .map((uri) => normalizeSlashes(uri.fsPath)),
     );
 }
+
+export async function TraverseTspConfigFileInWorkspace() {
```

Comment:
feels it's not needed to make a utility method for this which is actually one line code.